x-pack/test/functional/services/monitoring/cluster_overview.js

```diff
@@ -79,6 +79,10 @@ export function MonitoringClusterOverviewProvider({ getService }) {
       return testSubjects.click('alerts-modal-remind-later-button');
     }
 
+    acceptAlertsModal() {
+      return testSubjects.click('alerts-modal-button');
```

Comment:
Does something happen when this is clicked that we should wait for?  Something to go away or something to appear?  A loading indicator?  If there is, I'd prefer it be here instead of in a test that calls this method.

Reply:
It's just the modal that closes and installs the alerts so there's no loading indicator or anything like that.